wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -120,60 +122,60 @@ Hold `Alt` to trigger Distance Snap. The value can range from x0.1 - x2.0.
 
 | Button (Keyboard Shortcut) | Usage | Description |
 | -------------------------- | ----- | ----------- |
-| New Combo (`Q`) | **Right-click:** Change current note to next combo colour. | Note combo. This is equivalent to a verse / measure in song / instrument script. An acceptable range is about 5-20. |
+| New Combo (`Q`) | **Right-click:** Change the current note to the next combo colour. | Note combo. This is equivalent to a verse / measure in song / instrument script. An acceptable range is about 5-20. |
 
 #### Hit Sounds
 
 | Button (Keyboard Shortcut) | Usage | Description |
 | -------------------------- | ----- | ----------- |
-| Whistle (`W`) | ("Select" on a Hit note) **Left-Click:** Use whistle sound on this note | Add a whistle sound addition to the selection. |
-| Finish (`E`) | ("Select" on a Hit note) **Left-Click:** Use finish sound on this note | Add a finish (cymbal) sound addition to the selection. |
-| Clap (`R`) | ("Select" on a Hit note) **Left-Click:** Use clap sound on this note | Add a clap sound addition to the selection. |
+| Whistle (`W`) | ("Select" on a hit note) **Left-Click:** Add a whistle sound on this note | Add a whistle sound addition to the selection. |
+| Finish (`E`) | ("Select" on a hit note) **Left-Click:** Add a finish sound on this note | Add a finish (cymbal) sound addition to the selection. |
+| Clap (`R`) | ("Select" on a hit note) **Left-Click:** Add a clap sound on this note | Add a clap sound addition to the selection. |
```

Comment:
following shortcuts are missing
crtl+q/w/e/r change the additions in order of auto/normal/soft/drum being q/w/e/r
shift+q/w/e/r change sampleset of note